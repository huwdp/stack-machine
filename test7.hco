push 0
push 10
push 33
push 100
push 108
push 114
push 111
push 119
push 32
push 44
push 111
push 108
push 108
push 101
push 72


label loop
printAscii
pop
push 0
jn loop
