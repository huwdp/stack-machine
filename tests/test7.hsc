pushi 0
pushi 10
pushi 33
pushi 100
pushi 108
pushi 114
pushi 111
pushi 119
pushi 32
pushi 44
pushi 111
pushi 108
pushi 108
pushi 101
pushi 72


label loop
printa
pop
pushi 0
jn loop
