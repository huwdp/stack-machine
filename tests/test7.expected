Welcome to Huw's Stack Machine
Hello, world!
